### ISSUE TITLE
NavigationControl triggers hover on layer underneath

### ISSUE THREAD

**Issue by dottodot** [visgl/deck.gl#5095]
#### Description
I have a working NavigationControl setup on my map but the only issue is when the map is behind the buttons the onHover event gets triggered. I have set the z-index on MapNav to 1 and even tried a really high value but still have the same issue.

#### Repro Steps
```
const layers = [
    new GeoJsonLayer({
      id: 'geojson',
      data: geojson,
      opacity: 1,
      stroked: false,
      filled: true,
      extruded: true,
      wireframe: false,
      material: material,
      getElevation: f => f.properties.height * 1400,
      getFillColor: f => f.properties.color,
      updateTriggers: {
        getElevation: [plantType],
        getFillColor: [plantType]
      },
      getLineColor: [255, 255, 255],
      pickable: true,
      transitions: {
        getElevation: {
          duration: 500,
        },
        getFillColor: {
          duration: 500,
        }
      },
      onHover: info => setHoverInfo(info)
    }),
    new GeoJsonLayer({
      id: 'geojson_bg',
      data: bg,
      opacity: 1,
      stroked: false,
      filled: true,
      material: false,
      pickable: false,
      getFillColor: [255, 255, 255],
      getLineColor: [255, 255, 255],
    }),
  ];
```
```
      <DeckGL
        layers={layers}
        initialViewState={initialViewState}
        controller={true}
        onLoad={mapLoaded}
        onViewStateChange={viewStateChange}
        ContextProvider={MapContext.Provider}
      >
        <MapNav>
          <NavigationControl showCompass={false} />
        </MapNav>
      </DeckGL>
```

#### Environment (please complete the following information):
 - Framework Version: deck.gl 8.3.5
 - Browser Version: Chrome 86.0.4240.111
 - OS: MacOs 10.15.7

**Comment by Pessimistress** [comment]
Do you mean "triggers hover" in the title? I don't think click will be triggered with https://visgl.github.io/react-map-gl/docs/advanced/custom-components#captureclick

**Comment by dottodot** [comment]
Yes sorry I meant the hover event, on mobile it behaves like a click instead

I have tried ` <NavigationControl showCompass={false} captureClick={false} />` but that doesn't resolve the issue.

**Comment by Pessimistress** [comment]
You do want `captureClick: true` because that stops `onClick` from firing in the map.

I think to solve this `<NavigationControl>` needs to add a similar prop `capturePointerMove`.

**Comment by dottodot** [comment]
Unfortunately `captureClick: true` made no difference

**Comment by Pessimistress** [comment]
@dottodot You misread my comment. This requires a new feature in react-map-gl.

**Comment by dottodot** [comment]
@Pessimistress I see what you mean about `capturePointerMove` just gave it a try and it resolves the issue. I'll create a PR on react-map-gl

**Comment by Pessimistress** [comment]
This is addressed in `react-map-gl@5.2.10`:

```jsx
<NavigationControl capturePointerMove={true} />
```

Thank you for your contribution!
